packages/core/realtime-js/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 2.74.0 (2025-10-06)
+
+This was a version bump only for @supabase/realtime-js to align it with other projects, there were no code changes.
```

Comment:
Should we add the replay change?

Reply:
Yeah ideally this will be automated in the future. Let me fix this. Thanks!